spec/generators/pageflow/routes/routes_generator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require 'support/generators'
+
+# Generators are not automatically loaded by Rails
+require 'generators/pageflow/routes/routes_generator'
+
+module Pageflow
+  module Generators
+    describe RoutesGenerator, type: :generator do
+      # intentionally left empty since the routes generator depends on ActiveAdmin-generated routes.
```

Comment:
Why not also add a fixture file with the basic active admin routes? You also do this for the user model. Otherwise let's delete this file, since it does nothing.